Makefile

```diff
@@ -49,6 +49,9 @@ python-quality-test:
 
 run-tests: python-test python-quality-test
 
+run-integration-tests: install-dev-dependencies
```

Comment:
Nice! I think this is ok for now, once we've moved all the unit tests to their respective unit folders,we can change this to do something like pytest ./eox_core --ignore-glob='**/unit/*'